app/models/claims/school.rb

```diff
@@ -68,4 +72,8 @@ class Claims::School < School
   has_many :claims
   has_many :mentor_memberships
   has_many :mentors, through: :mentor_memberships
+
+  def grant_conditions_accepted_by_user
+    User.find_by_id(grant_conditions_accepted_by_id)&.full_name
+  end
```

Comment:
I think it'll be cleaner to do the following:

```suggestion
  
  belongs_to :grant_conditions_accepted_by, class_name: "User", optional: true
```

Then

```ruby
school.grant_conditions_accepted_by&.full_name
```